src/pool/srv_target.c

```diff
@@ -592,17 +598,18 @@ ds_pool_start(uuid_t uuid)
 			       NULL /* create_args */, &llink);
 	if (rc == 0) {
 		pool = pool_obj(llink);
-		if (pool->sp_stopping)
-			/* Restart it and hold the reference. */
-			pool->sp_stopping = 0;
-		else
-			/* Already started; drop our reference. */
-			daos_lru_ref_release(pool_cache, &pool->sp_entry);
-		goto out_lock;
+		if (pool->sp_stopping) {
+			D_ERROR(DF_UUID": stopping isn't done yet\n",
+				DP_UUID(uuid));
+			rc = -DER_BUSY;
```

Comment:
This seems changing the behavior.   Do we understand why it just reset sp_stopping in the original implementation.

Reply:
Yes, this should be okay.